Apps/Admin/Client/Components/Support/Covid19ImmunizationSection.razor

```diff
@@ -1,7 +1,7 @@
 @using HealthGateway.Common.Data.Utils
 @inherits Fluxor.Blazor.Web.Components.FluxorComponent
 
-<MudText Class="mt-4" Typo="Typo.subtitle1">
+<MudText Class="mt-4" Typo="Typo.h6">
     COVID‑19 Immunizations
```

Comment:
If there's going to be another "COVID-19" label immediately above this, I'd suggest renaming to just "Immunizations".

Reply:
The change in typography also makes this header a different size from "Assessment History", which feels off.